src/System.Management.Automation/engine/Modules/ModuleCmdletBase.cs

```diff
@@ -256,6 +261,11 @@ internal List<WildcardPattern> MatchAll
             "ModuleVersion"
         };
 
+        internal static IReadOnlyList<string> DefaultCompatiblePSEditions { get; } = new string[]
```

Comment:
This is the default if CompatiblePSEditions isn't declared in the module manifest, correct?  Should have a comment to clarify this as someone unfamiliar with the intent may thing this is what PSCore6 is by default compatible with.